test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpErrorTest/HandlesQuotesAfterTransition.stree.txt

```diff
@@ -1,5 +1,8 @@
-Expression block - Gen<Expr> - 1 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    Code span - Gen<Expr> - [] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (1:0,1) - Tokens:1
-        SyntaxKind.Unknown;[];
+CSharpCodeBlock - [0..1)::1 - [@]
```

Comment:
It might be valuable to indicate how each of these tests were parsed. Aka, with a document parse or a csharp parse. Granted, if we eventually move everything to parse by document then this wont be an issue.